### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by leafyao8621** [calliebusch/ORIE_4741#11]
This project predicts time durations of the Citi bike trips as well as classifies outliers in the time durations. Using data directly from Citi bike and external weather data, this project constructs several models, both regression and classification to achieve its goal.

For the regression part, the pre-analysis of the performance of different algorithms provides a good overview, which gets proved by the actual analysis.  The comparison between different algorithms is good. And hyperparameter tuning does improve the overall performance.

For the classification part, it can be better if a different metric is used. From the confusion matrix, the false-negative rate is significantly higher than the false-positive rate. Hyper-parameter tuning with a different metric can be used to get better overall performance.

Overall, there should have been some analysis of generalization, be it simple train-test split or cross-validation. It is especially important as hyperparameter tuning is used to get a higher training accuracy which may well lead to overfitting and poor generalization.
